docs/page-objects/inputs.rst

```diff
@@ -109,6 +119,30 @@ There are also methods to select content from responses: :meth:`jmespath()
     [<Selector query='//h1/text()' data='Title'>]
 
 
+.. _browserresponse:
+
+Working with BrowserResponse
+=============================
+
+:class:`~.BrowserResponse` is similar to :class:`~.HttpResponse`, but for
+browser-rendered pages. It has an :attr:`~.BrowserResponse.html` attribute
+instead of a :attr:`~.HttpResponse.text` attribute, which contains the rendered
```

Comment:
It now has .text attribute. I think that's fine to document html-first, but "instead of" is not 100% accurate now. It seems fine to just document it on its own, and drop many comparisons with HttpResponse (mentioning HttpResponse is still good).